src/components/environment-variables/environment-variables-list.js

```diff
@@ -111,7 +114,7 @@ const EnvironmentVariablesList = (props) => {
         </div>
       )}
 
-      {saveState && saveState.err && (
+      {saveState?.error && (
```

Comment:
`saveState.err` same as `saveState.error`?

Reply:
saveState.err does not exist on this variable

Reply:
nice catch 👍